src/relay/transforms/simplify_expr.cc

```diff
@@ -585,6 +585,35 @@ class EliminateIdentityRewrite : public DFPatternRewrite {
   DFPattern const_;
 };
 
+/*! \brief Make two consecuitive add able to be constant_folded. */
+class SimplifyConsecuitiveAdd : public DFPatternRewrite {
+ public:
+  SimplifyConsecuitiveAdd() {
+    x_ = IsWildcard();
+    const1_ = IsConstant();
+    const2_ = IsConstant();
+    DFPattern add_op_ = IsOp("add");
+    pattern_ = add_op_({add_op_({x_, const1_}), const2_});
```

Comment:
the pattern will not be selected if any input of the add op is non-const, right?

Reply:
It only works when two add operators have one same input and two const.